frappe/templates/includes/blog/blog.html

```diff
@@ -5,25 +5,6 @@
 {% block hero %}{% endblock %}
 
 {% block page_content %}
-<style>
```

Comment:
Put these styles in website.less

Reply:
Thanks for the review I added the styles to website.less and also tweak the styles a bit for adding spacing around avatar and blog posts